CODEOWNERS

```diff
@@ -1,8 +1,79 @@
-# Beatriz Rizental as main contact for addon related changes
-/addons/ @brizental
-/src/addons/ @brizental
-/signature/ @brizental
+# These owners will be the default owners for each directory
+# in the repo.
 
-# Francesco Lodolo as main contact for string changes
+/addons/ @brizental @bakulf @MattLichtenstein
 addons/**/manifest.json @flodolo
+
+/3rdparty/adjust-android-sdk/ @bakulf @birdsarah
+/3rdparty/adjust-ios-sdk/ @bakulf @birdsarah
+/3rdparty/glean/ @brizental
+/3rdparty/openSSL/ @strseb
+/3rdparty/sentry/ @strseb
+/3rdparty/wireguard-apple/ @bakulf @birdsarah @mcleinman
+/3rdparty/wireguard-go/ @bakulf @oskirby
+/3rdparty/wireguard-tools/ @bakulf @oskirby
+
+/android/ @strseb
+/balrog/ @bakulf @birdsarah
+/ci_scripts/ @birdsarah
+/docs/ @birdsarah
+/extension/ @bakulf @birdsarah
+/glean/ @brizental
+/i18n\ *\ 82b6178/ @flodolo
+/ios/ @bakulf @birdsarah @mcleinman
+/lottie/ @bakulf @birdsarah
+/macos/ @bakulf @birdsarah
+/nebula/ @MattLichtenstein @lesleyjanenorton @flozia
+/scripts/ @bakulf @oskirby
+/signature/ @bakulf @birdsarah @brizental
+
+/src/ @birdsarah
+/src/addons/ @brizental @bakulf
+/src/adjust/ @bakulf @birdsarah
+/src/authenticationinapp/ @bakulf @birdsarah
+/src/captiveportal/ @birdsarah
+/src/cmake/ @oskirby
+/src/commands/ @bakulf @birdsarah
+/src/composer/ @bakulf @brizental
+/src/connectionbenchmark/ @bakulf @flozia
+/src/crashreporter/ @strseb
+/src/daemon @birdsarah
+/src/frontend/ @lesleyjanenorton @MattLichtenstein @flozia
+src/glean/ @brizental
+/src/hacl-star/ @bakulf @birdsarah
+/src/inspector/ @bakulf @oskirby @strseb
+/src/mfbt/ @bakulf @birdsarah
+/src/models/ @bakulf @birdsarah
+/src/platforms/android/ @strseb
+src/platforms/dummy/ @bakulf @birdsarah
```

Comment:
I know that dummy folder doesn't currently have a subfolder, but no harm in preceding this with /